src/pages/iv-category-levels/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect } from 'react'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import * as yup from 'yup'
+import { useContext } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import toast from 'react-hot-toast'
+import { useResourceQuery } from 'src/hooks/resource'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useForm } from 'src/hooks/form'
+import { useError } from 'src/error'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+
+const CategoryLevels = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack: stackError } = useError()
+
+  const { labels: labels, access } = useResourceQuery({
+    datasetId: ResourceIds.CategoryLevels
+  })
+
+  useEffect(() => {
+    getGridData()
+  }, [])
+
+  function getGridData() {
+    getRequest({
+      extension: InventoryRepository.CategoryLevel.qry
+    })
+      .then(res => {
+        const processedData = res.list.map((item, index) => ({
+          id: index + 1,
+          ...item
+        }))
+        formik.setValues({ items: processedData })
+      })
+      .catch(error => {})
```

Comment:
as rebecca requested we should stop using catch and try 
no need anymore for them 
they are internally implemented